.github/workflows/main.yaml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Was this intentional?

Reply:
yeah I changed it to be consistent with our development environment and so that we don't need to add conditional dependencies to make linux builds work (which we previously had).